src/core/messaging/ACSClient.ts

```diff
@@ -280,6 +280,13 @@ export class ACSConversation {
 
         try {
             const listener = (event: TypingIndicatorReceivedEvent) => {
+                const {sender, recipient} = event;
+
+                // Ignore participant's own typing events
+                if ((sender as any).communicationUserId === (recipient as any).communicationUserId) { // eslint-disable-line @typescript-eslint/no-explicit-any

```

Comment:
do we assume that those objects are always present including communicationUserId ?

Reply:
Yes since it's an ACS typing event and should always contain a sender & recipient with its id.